### DISCUSSION TITLE
As a premium user, how do I get Version 8.0 released two days ago?

### DISCUSSION THREAD

**Discussion by millysecurity** [Hyperclaw79/PokeBall-SelfBot#213] [Q&A] [answered]
I've messaged Hyperclaw a couple of times  on the 11th and 12th (feature requests/troubleshooting) and gotten no responses. Haven't seen you online in a while. Any chance you can shoot me over the new version if you see this? - Milly

**Comment by Hyperclaw79** [comment, marked as answer]
Hey Milly, I might've overlooked the DMs. I'll send you the new links on
Discord.

**Reply by millysecurity** [reply]
No worries thanks for the quick response! Anyone reading this just want yall to know this dude is actually skilled at programming so you're gonna be getting your money's worth. Anyways thanks again!

**Comment by Hyperclaw79** [comment]
Thanks Milly ❤️
